test/cpp_alias_template.cpp

```diff
@@ -85,11 +87,19 @@ using h = g<T, a>;
         else if (alias.name() == "d")
         {
             check_template_parameters(alias, {{cpp_entity_kind::template_type_parameter_t, "T"}});
-
+            #ifndef CPPAST_TEMPLATE_FULLARGUMENTSPARSING
             cpp_template_instantiation_type::builder builder(
                 cpp_template_ref(cpp_entity_id(""), "a"));
-            builder.add_unexposed_arguments("void");
-            REQUIRE(equal_types(idx, alias.type_alias().underlying_type(), *builder.finish()));
+
+                builder.add_unexposed_arguments("void");
+                REQUIRE(equal_types(idx, alias.type_alias().underlying_type(), *builder.finish()));
+            #else                         
+            // need to write the code for full argument parsing using add_argument
```

Comment:
What do you mean by this comment?

Reply:
I didn't have the needed time to write the test for full argument parsing.